canonical-web-and-design/ubuntu.com
# Remove old advantage

## Done

- Remove the templates and javascript for the old subscriptions table.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10441.demos.haus/advantage?test_backend=true).
- Check that the page looks fine both log in and out.
- Visit [/advantage/subscribe?test_backend=true](https://ubuntu-com-10441.demos.haus/advantage/subscribe?test_backend=true) and make sure clicking "Buy now" opens the modal.


## Issue / Card

Fixes: https://github.com/canonical-web-and-design/commercial-squad/issues/122.
